### ISSUE TITLE
MeanStdScaling is a bad name

### ISSUE THREAD

**Issue by glennmoy** [invenia/FeatureTransforms.jl#87]
As noted by @eperim 

>"Standardised" data is a pretty common nomenclature, while "MeanStdScaling" sounds like it could be anything (and the data is not even scaled by the mean, it is shifted by it, only the std is used for scaling).

**Comment by bencottier** [comment]
I think it's fine! "Standardised" sounds to me like it could be many things 😅 With `MeanStdScaling` I know immediately what it means among all the common ways of scaling data (and I thought so before we wrote this Transform).

I do think "Standardised" is better than "Normalised" (which I considered originally) because while that has "Normal" in the name, it can refer to scaling by any kind of norm.

**Comment by eperim** [comment]
I disagree. "Standardisation" is quite common nomenclature. There's even a [special tag for it in Stats Exchange](https://stats.stackexchange.com/questions/tagged/standardization), and you can find the term frequently in [glossaries](https://statisticsbyjim.com/glossary/standardization/), [multiple](https://kharshit.github.io/blog/2018/03/23/scaling-vs-normalization) [blog posts](https://www.statisticshowto.com/standardized-values-examples/), [teaching materials](https://amsi.org.au/ESA_Senior_Years/SeniorTopic4/4h/4h_2content_8.html), and in other packages (e.g. [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html)). These are all from the first page of a Google search.

> With MeanStdScaling I know immediately what it means among all the common ways of scaling data in ML

Naturally you do, because you chose the name, but is it common nomenclature that you expect people in general to be familiar with?

**Comment by bencottier** [comment]
> "Standardisation" is quite common nomenclature.

Thanks for gathering those links, that makes me favour "Standardisation" more.

> Naturally you do, because you chose the name, but is it common nomenclature that you expect people in general to be familiar with?

I copied the name from FeatureEngineering, but yes I had the chance to rename in the original PR and decided to keep it. I think more in terms of "how to identify what this literally does with a relatively short name?" than the nomenclature. You might even disagree that "MeanStdScaling" is best for that purpose, but that's my reasoning.

I commented to show there's a difference of opinion, but I don't feel strongly about this. Nomenclature is important and it's very reasonable to rename it.

**Comment by molet** [comment]
I would also vote for "Standardization" instead of "MeanStdScaling".
Also, would it be possible to add "Normalization" as well to the package?
I.e. ( x - min(x) ) / ( max(x) - min(x) )

**Comment by xiaodaigh** [comment]
MeanStd is more precise

**Comment by mzgubic** [comment]
How does [StandardScaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html) sound?
